### ISSUE TITLE
"Rich text limited" text format should support aria-labels for links (for phone numbers)

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#3260]
**Describe the defect**

`aria-label` attribute added by editors gets stripped from `a` tags.

**To Reproduce**
1. Go to https://staging.cms.va.gov/node/8420/edit 
1. Edit the source of the Answer rich text field, and paste in this link `<a href="tel:+18008271000" aria-label="8 0 0. 8 2 7. 1 0 0 0.">800-827-1000</a>`
1. Uncheck "Source"
1. Check source again, aria-label has disappeared

**Expected behavior**
`aria-label` attribute should be retained.

**Screenshots**

![Compose_tips___Veterans_Affairs](https://user-images.githubusercontent.com/643678/96872956-48fefe80-1442-11eb-9d7a-6e5b386729bc.jpg)

## AC

- [ ] compare allowed html attributes for Rich Text and Rich Text Limited formats and ensure that all relevant attributes exist in Rich Text Limited format settings

we dont want to transfer `class` attr.
